packages/elements/src/components/ino-icon/icons.js

```diff
@@ -1,9 +1,5 @@
 export default [
-  '_fab_set_arrow_down',
```

Comment:
Why did the order change? Did you use the `integrate:icons` script or did you change it manually?

Reply:
I used the `integrate:icons` script. Maybe this "issue" depens on my operating system (Windows)?

Reply:
Could be, but doesn't really matter